e2e/commands/db.ts

```diff
@@ -0,0 +1,17 @@
+import { Page } from 'puppeteer'
+
+export function clean(page: Page, dbName: string, storeName: string) {
+    return new Promise((resolve, reject) => {
+        const request = indexedDB.open(dbName)
+
+        request.onsuccess = () => {
```

Comment:
you can use library `idb` to make it more clear

Reply:
Removed.